CGPs/0020.md

```diff
@@ -0,0 +1,36 @@
+# CGP [0020]: Update randomnessBlockRetentionWindow to Extend Attestation Expiration Duration
+
+- Date: 2021-01-29
+- Author(s): @anna-carroll
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+- Discussions To: [TODO: create and add discussion link]
```

Comment:
You'd probably wanna create a forum post and link it here

Reply:
cool, the link on the forum post will be broken until we merge this to master https://github.com/celo-org/celo-proposals/pull/155/commits/cdb1b17ac98e25b628014d5fbd34e6a1b7c5bdfb